generated_types.go

```diff
@@ -267,6 +269,18 @@ type AssessmentPlan struct {
 	// Enforce that control reference and the assessments' references match
 	// This formulation uses the control's reference if the assessment doesn't include a reference
 	Assessments []Assessment `json:"assessments" yaml:"assessments"`
+
```

Comment:
Realized this is getting merged with the Layer 4 Assessment Plan struct. I think we could either rename or just remove the Layer 4 Evaluation Plan logic here.

Reply:
Exploring Option B here - https://github.com/eddie-knight/gemara/pull/4